native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.ios.js

```diff
@@ -132,10 +132,11 @@ const styles = StyleSheet.create({
     paddingTop: (spacingXl - 2 * borderSizeSm - spacingBase) / 2,
   },
   buttonExpand: {
+    paddingLeft: spacingMd,
     color: colorGray700,
   },
   buttonClose: {
-    color: colorGray500,
+    color: colorGray700,
```

Comment:
@willhg pointed out we have two different colours for the close icon and the chevron, this binrgs balance to the force